src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -461,6 +480,18 @@ function CreatePipelineRun() {
             defaultMessage: 'Create PipelineRun'
           })}
         </h1>
+        <div className="tkn--create--yaml-mode">
+          <Button
+            kind="tertiary"
+            id="format-pipeline-button"
+            onClick={switchToYamlMode}
+          >
+            {intl.formatMessage({
+              id: 'dashboard.createPipelineRun.mode.yaml',
```

Comment:
`dashboard.createPipelineRun.yamlModeButton`
Makes it clearer that it's used for an action element which is important for some languages where actions and titles can be translated differently.